parm/postcntrl_hafs.xml

```diff
@@ -92,16 +92,8 @@
        </param>
 
        <param>
-       <shortname>TKE_ON_ISOBARIC_SFC</shortname>
-       <pname>TKE</pname>
-       <level>200. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 
-47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.</level>
-       <scale>3.0</scale>
-       </param>
-
-       <param>
-       <shortname>CLMR_ON_ISOBARIC_SFC</shortname>
-       <pname>CLMR</pname>
+       <shortname>CLWMR_ON_ISOBARIC_SFC</shortname>
```

Comment:
@LinZhu-NOAA Can you list the variables removed and added in hafs products in the issue #366?

Reply:
Sure. I will list them in the issue #366 .

Reply:
The variables added in and removed from hafs products are listed in the issue #366.